cpu/qn908x/include/periph_cpu.h

```diff
@@ -366,6 +366,7 @@ typedef struct {
     gpio_t pin_scl;         /**< SCL pin */
     gpio_t pin_sda;         /**< SDA pin */
     uint32_t speed;         /**< bus speed in bit/s */
+    bool external_pull_up;  /**< Do not enable internal pull-up on SCL? */
```

Comment:
Does anything speak against always enabling this?
That's now it's done on e.g. nRF5x